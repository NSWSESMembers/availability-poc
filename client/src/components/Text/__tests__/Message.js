import React from 'react';
import renderer from 'react-test-renderer';

import { Message } from '../';

test('renders correctly', () => {
  const tree = renderer.create(
    <Message>Test</Message>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
