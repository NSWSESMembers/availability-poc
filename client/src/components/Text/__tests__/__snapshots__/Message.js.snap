// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "flexDirection": "row",
        "justifyContent": "center",
      },
    ]
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={undefined}
  >
    Test
  </Text>
</View>
`;
